### PR TITLE
[APITESTS] Tested function must be called before GetLastError() log

### PR DESCRIPTION
**Pull request by SergeGautherie**
To get the relevant `LastError` value.

### CODE REVIEW

File: modules/rostests/apitests/kernel32/GetComputerNameEx.c [hunk 1]
```diff
@@ -418,9 +444,14 @@ TestReturnValues()
     StartSeh()
         GetComputerNameExA(ComputerNameDnsDomain, ComputerName, &ComputerNameSize);
         if (ComputerNameSize)
-           ok(GetComputerNameExA(ComputerNameDnsDomain, ComputerName, &ComputerNameSize), "GetComputerNameExA(ComputerNameDnsDomain) failed with %ld\n", GetLastError());
+        {
+           ret = GetComputerNameExA(ComputerNameDnsDomain, ComputerName, &ComputerNameSize);
```
Comment (ThFabba) [inline modules/rostests/apitests/kernel32/GetComputerNameEx.c]: Indent problem

File: modules/rostests/apitests/kernel32/GetComputerNameEx.c [hunk 2]
```diff
@@ -328,6 +328,7 @@ TestReturnValues()
     CHAR ComputerName[128];
     DWORD ComputerNameSize = 0;
     INT ErrorCode;
+    BOOL ret;
```
Comment (ThFabba) [inline modules/rostests/apitests/kernel32/GetComputerNameEx.c]: Can you name this `Ret` please to be consistent with the rest of the file?

File: modules/rostests/apitests/kernel32/PrivMoveFileIdentityW.c [hunk 3]
```diff
@@ -97,13 +98,15 @@ TestPrivMoveFileIdentityW(VOID)
...
+    ret = pPrivMoveFileIdentityW(Self, FileName, 0);
+    ok(!ret, "PrivMoveFileIdentityW succeeded unexpectedly\n");
     ok(GetLastError() == ERROR_SHARING_VIOLATION, "Last error: %#lx\n", GetLastError());
     ok(QueryFileInfo(FileName, &FileBasicInfo, &FileStandardInfo) == TRUE, "QueryFileInfo returned FALSE\n");
     ok(FileBasicInfo.CreationTime.QuadPart != CreationTime.QuadPart, "Equal creation times\n");
     ok(FileStandardInfo.EndOfFile.QuadPart == 0LL, "File wasn't created empty: %I64d\n", FileStandardInfo.EndOfFile.QuadPart);
     SetLastError(0xdeadbeef);
-    ok(pPrivMoveFileIdentityW(Self, FileName, 2) == TRUE, "PrivMoveFileIdentityW failed with %#lx\n", GetLastError());
+    ret = pPrivMoveFileIdentityW(Self, FileName, 2);
+    ok(ret, "PrivMoveFileIdentityW failed with %#lx\n", GetLastError());
```
Comment (ThFabba) [inline modules/rostests/apitests/kernel32/PrivMoveFileIdentityW.c]: Keep the `== TRUE`

File: modules/rostests/apitests/kernel32/PrivMoveFileIdentityW.c [hunk 4]
```diff
@@ -60,6 +60,7 @@ TestPrivMoveFileIdentityW(VOID)
     HANDLE hDest;
     WCHAR Self[MAX_PATH];
     OFSTRUCT ReOpen;
+    BOOL ret;
```
Comment (ThFabba) [inline modules/rostests/apitests/kernel32/PrivMoveFileIdentityW.c]: Arguably `Ret` suits the surrounding style here more as well.

File: modules/rostests/apitests/kernel32/GetComputerNameEx.c (hunk 2 above)
Comment (SergeGautherie) [reply modules/rostests/apitests/kernel32/GetComputerNameEx.c]: Done.

File: modules/rostests/apitests/kernel32/GetComputerNameEx.c (hunk 1 above)
Comment (SergeGautherie) [reply modules/rostests/apitests/kernel32/GetComputerNameEx.c]: Fixed.

File: modules/rostests/apitests/kernel32/PrivMoveFileIdentityW.c (hunk 4 above)
Comment (SergeGautherie) [reply modules/rostests/apitests/kernel32/PrivMoveFileIdentityW.c]: Done.

File: modules/rostests/apitests/kernel32/PrivMoveFileIdentityW.c (hunk 3 above)
Comment (SergeGautherie) [reply modules/rostests/apitests/kernel32/PrivMoveFileIdentityW.c]: Reverted.

**Pull request merged**